compiler-rt/lib/rtsan/rtsan_interceptors.cpp

```diff
@@ -39,7 +39,6 @@
 
 using namespace __sanitizer;
 
-using __rtsan::rtsan_init_is_running;
 using __rtsan::rtsan_initialized;
```

Comment:
It would be better to use atomics here
see `static atomic_uint8_t asan_inited`

Reply:
Fixed in this latest version. Thanks!